### PR TITLE
adding periodic e2e job to test windows containerd azuredisk csi driver

### PR DESCRIPTION
**Pull request by marosset**

### CODE REVIEW

File: config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml [hunk 1]
```diff
@@ -404,6 +404,66 @@ periodics:
...
+      - --down
+      - --build=quick
+      - --dump=$(ARTIFACTS)
+      # Azure-specific test args
+      - --deployment=aksengine
+      - --provider=skeleton
+      - --aksengine-admin-username=azureuser
+      - --aksengine-admin-password=AdminPassw0rd
+      - --aksengine-creds=$(AZURE_CREDENTIALS)
+      - --aksengine-download-url=https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz
```
Comment (chewong) [inline config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml:446]: `https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz` is using aks-engine v0.48.0. Do we need to update the version?

Comment (marosset) [reply config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml:446]: hmm, let me check, I think we need a version newer than v0.48.0. Maybe I'll just point it to more recent release on github.

**Comment by benmoss** [comment]
@marosset looks like the upstream PR has merged

File: config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml (hunk 1 above)
Comment (chewong) [reply config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml:446]: Updated aks-engine version to v0.53.0.

**Pull request merged**